src/main.cpp

```diff
@@ -100,77 +100,77 @@ base::Vector parsePlayerPosition(const std::string& playerPosString) {
+  //   ("play-level,l",
+  //    po::value<std::string>(),
+  //    "Directly jump to given map, skipping intro/menu etc.")
+  //   ("player-pos",
+  //    po::value<std::string>(),
+  //    "Specify position to place the player at (to be used in conjunction with\n"
+  //    "'play-level')")
+  //   ("debug-mode,d",
+  //    po::bool_switch(&config.mDebugModeEnabled),
+  //    "Enable debugging features")
+  //   ("game-path",
+  //    po::value<std::string>(&config.mGamePath)->default_value(""),
+  //    "Path to original game's installation. Can also be given as positional "
+  //    "argument. If not provided here, a folder browser ui will ask for it");
+
+  // po::positional_options_description positionalArgsDescription;
+  // positionalArgsDescription.add("game-path", -1);
 
   try
   {
-    po::variables_map options;
-    po::store(
-      po::command_line_parser(argc, argv)
-        .options(optionsDescription)
-        .positional(positionalArgsDescription)
-        .run(),
-      options);
-    po::notify(options);
-
-    if (options.count("help")) {
-      std::cout << optionsDescription << '\n';
-      return 0;
-    }
-
-    if (options.count("play-level")) {
-      config.mLevelToJumpTo =
-        parseLevelToJumpTo(options["play-level"].as<std::string>());
-    }
-
-    if (options.count("player-pos")) {
-      if (!options.count("play-level")) {
-        throw std::invalid_argument(
-          "This option requires also using the play-level option");
-      }
-
-      config.mPlayerPosition = parsePlayerPosition(
-        options["player-pos"].as<std::string>());
-    }
-
-    if (!config.mGamePath.empty() && config.mGamePath.back() != '/') {
-      config.mGamePath += "/";
-    }
+  //   po::variables_map options;
+  //   po::store(
+  //     po::command_line_parser(argc, argv)
+  //       .options(optionsDescription)
+  //       .positional(positionalArgsDescription)
+  //       .run(),
+  //     options);
+  //   po::notify(options);
+
+  //   if (options.count("help")) {
+  //     std::cout << optionsDescription << '\n';
+  //     return 0;
+  //   }
+
+  //   if (options.count("play-level")) {
+  //     config.mLevelToJumpTo =
+  //       parseLevelToJumpTo(options["play-level"].as<std::string>());
+  //   }
+
+  //   if (options.count("player-pos")) {
+  //     if (!options.count("play-level")) {
+  //       throw std::invalid_argument(
+  //         "This option requires also using the play-level option");
+  //     }
+
+  //     config.mPlayerPosition = parsePlayerPosition(
+  //       options["player-pos"].as<std::string>());
+  //   }
+
+  //   if (!config.mGamePath.empty() && config.mGamePath.back() != '/') {
+  //     config.mGamePath += "/";
+  //   }
 
     gameMain(config);
   }
-  catch (const po::error& err)
-  {
-    std::cerr << "ERROR: " << err.what() << "\n\n";
-    std::cerr << optionsDescription << '\n';
-    return -1;
-  }
+  // catch (const po::error& err)
```

Comment:
I think we should have a separate `emscripten_main.cpp` file which is not using `program_options`, and then we can add either `main.cpp` or `emscripten_main.cpp` via CMake. That way, we don't need so much conditional compilation.